theano/gpuarray/basic_ops.py

```diff
@@ -263,6 +263,9 @@ def m(t):
 def get_ctype(dtype):
     if dtype is gpuarray.GpuArray:
         return "gpudata *"
+    elif isinstance(dtype, numpy.dtype):
+        # The if is needed as numpy.dtype can't be compared to gpuarray.[S]SIZE
+        return 'npy_' + dtype.name
```

Comment:
Why do you say numpy.dtype can't be compared?  I never had any problems with this before.